src/blocks/form/edit.js

```diff
@@ -85,6 +85,8 @@ export const childBlocks = [
 			icon: icons.name,
 			supports: {
 				inserter: false,
+				html: false,
```

Comment:
We could move these to the `FieldDefaults` const up above (along with `inserter: false`) to clean this up, instead of repeating it each time. The entire `supports` could be part of `FieldDefaults`.

Reply:
FieldDefaults now has the correct supports. Removed supports from individual children blocks. 
```javascript
supports: {
		reusable: false,
		html: false,
		inserter: false,
	},
```